### ISSUE TITLE
Argument value of 'false' is initialized to 'nil'

### ISSUE THREAD

**Issue by chrisbutler** [pawurb/smart_init#2]
In the following example:
```rb
class TestExample < SmartInit::Base
  initialize_with :test_value
  is_callable

  def call
    SomeItems.where(value: test_value)
  end
end

TestExample.call(test_value: false)
```

The ActiveRecord query ends up being `where(value: nil)` which - for this example - will result in the wrong records being returned.

Seems the issue might be in this falsey check?https://github.com/pawurb/smart_init/blob/3e5008b32b8df634356a73d57b49db56b0de64ed/lib/smart_init/main.rb#L31

**Comment by pawurb** [comment]
Thanks for spotting. It's now fixed and released as 3.5.0

**Comment by chrisbutler** [comment]
⚡️ awesome, thank you for such a quick response!
